test/verifiedTests/base/testIO/testUtilities/testOutputHypergraph.m

```diff
@@ -0,0 +1,30 @@
+% The COBRAToolbox: testOutputHypergraph.m
+%
+% Purpose:
+%     - test the outputHypergraph function
+%
+% Authors:
+%     - Jacek Wachowiak
+global CBTDIR
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testOutputHypergraph'));
+cd(fileDir);
+
+% test variables
+testModel = readCbModel([CBTDIR filesep 'test' filesep 'models' filesep 'ecoli_core_model.mat']);
+testFileName = 'testData_outputHypergraph.txt';
+testWeights = zeros(95);
+
+% function outputs
+outputHypergraph(testModel, testWeights, testFileName);
+testsFile = textread('testData_outputHypergraph.txt');
+modelFile = textread('refData_outputHypergraph.txt');
+
+% test
+assert(isequal(testsFile, modelFile))
+
```

Comment:
@jacekwachowiak, could you delete the `testData_outputHypergraph.txt` file at the end of your test?